packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -280,6 +283,30 @@ class EditableText extends StatefulWidget {
   /// The text style to use for the editable text.
   final TextStyle style;
 
+  /// {@template flutter.widgets.editableText.strutStyle}
+  /// The strut style used for the vertical layout.
+  ///
+  /// Strut is used to establish a predictable vertical layout. Since fonts may
+  /// vary depending on user input and due to font fallback, [StrutStyle.forceStrutHeight]
+  /// is enabled by default to lock all lines to the height of the base [TextStyle], provided by
+  /// [style]. This ensures the typed text fits within the alotted space.
+  ///
+  /// If null, the strut will is inherit values from the [style] and has
+  /// [StrutStyle.forceStrutHeight] set to true. When no [style] is passed,
```

Comment:
Is this second sentence just repeating the point made in the first one?

Reply:
The second sentence is about the way the theme in particular. It is talking about when [style] is null, which is a separate case than when [strutStyle] is null (lots of things named ___style here)